charts/emissary-ingress/templates/ambassador-agent.yaml

```diff
@@ -5,20 +5,16 @@
   {{- /* Case 1: "fullImageOverride" means don't bother checking the tag. */ -}}
   {{- /* Case 2: Otherwise, if it's not a semver-style version number,    */ -}}
   {{- /*         assume we have a power user and turn the agent on.       */ -}}
-  {{- /* Case 3: Otherwise, if Edge Stack, we need at least 1.12.0.       */ -}}
-  {{- /* Case 4: Otherwise, it's OSS and we need at 1.13.0.               */ -}}
+  {{- /* Case 3: Otherwise, it's OSS and we need at 1.13.0.               */ -}}
```

Comment:
`s/at 1.130/at least 1.13.0` ... and yeah, pretty sure that one was originally my bug. 😛 😂

Reply:
hmm wonder if we can just get rid of this for hte emissary chart.